plugins/explore-backend/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@backstage/plugin-explore-backend",
+  "version": "0.1.0",
```

Comment:
New packages should start at `0.0.0` and be accompanied by a `minor` bump in a changeset.